src/SDK/Language.php

```diff
@@ -97,4 +97,25 @@ protected function toCamelCase($str): string
         $str = str_replace(" ", "", $str);
         return lcfirst($str);
     }
+
+    public function getEnumType(string $service, string $method): string
+    {
+        switch ($service) {
```

Comment:
We should avoid Appwrite specific things. If we need namespaces they should either be auto-built or provided by Appwrite's spec.

Reply:
Its been removed and shifted to appwrite spec file (https://github.com/appwrite/appwrite/pull/5726)